randovania/games/fusion/logic_database/Main Deck.txt

```diff
@@ -1087,7 +1090,7 @@ Extra - room_id: [38]
   > Beside Door
       All of the following:
           Morph Ball and Screw Attack
-          # Get up
+          # Get up - https://youtu.be/ycp5oCd5-9I
```

Comment:
The video is the wrong place here as it shows neither WJ beginner not Can Climb High Jump.
You probably want to do:
```
ALL of the following:
# Get up
- Morph Ball and Screw Attack
- ANY of the following:
  - Wall Jump (Beginner)
  - Can Climb High Jump Wall
  - ALL of the following:
    # Screw only: https://you blabla 
  
    - Movement (Hypermode? unsure if that or expert)